api/terraform/catalogue_api/main.tf

```diff
@@ -1,33 +1,61 @@
-module "services" {
-  source = "services"
-
-  namespace = "${var.namespace}"
+module "service" {
+  source = "service"
 
+  namespace    = "${var.namespace}-${var.environment}"
   namespace_id = "${aws_service_discovery_private_dns_namespace.namespace.id}"
 
   subnets      = ["${var.subnets}"]
   cluster_name = "${var.cluster_name}"
   vpc_id       = "${var.vpc_id}"
-  nlb_arn      = "${module.nlb.arn}"
-
-  container_port = "${var.container_port}"
+  lb_arn       = "${var.lb_arn}"
 
-  remus_container_image = "${var.remus_container_image}"
-  remus_es_config       = "${var.remus_es_config}"
-  remus_listener_port   = "${local.remus_listener_port}"
+  container_port = "${local.api_container_port}"
 
-  romulus_container_image = "${var.romulus_container_image}"
-  romulus_es_config       = "${var.romulus_es_config}"
-  romulus_listener_port   = "${local.romulus_listener_port}"
+  container_image = "${var.api_container_image}"
+  es_config       = "${var.es_config}"
+  listener_port   = "${var.listener_port}"
 
   nginx_container_image = "${var.nginx_container_image}"
-  nginx_container_port  = "${var.nginx_container_port}"
+  nginx_container_port  = "${local.nginx_container_port}"
+
+  task_desired_count = "${var.task_desired_count}"
 
-  remus_task_number   = "${var.remus_task_number}"
-  romulus_task_number = "${var.romulus_task_number}"
+  security_group_ids = ["${var.lb_ingress_sg_id}"]
+
+  service_egress_security_group_id = "${aws_security_group.service_egress_security_group.id}"
 }
 
 resource "aws_service_discovery_private_dns_namespace" "namespace" {
-  name = "${var.namespace}"
+  name = "${var.namespace}-${var.environment}"
   vpc  = "${var.vpc_id}"
 }
+
+module "gateway_stage" {
+  source = "git::https://github.com/wellcometrust/terraform.git//api_gateway/modules/stage?ref=v17.0.0"
+
+  stage_name = "${var.environment}"
+  api_id     = "${var.api_id}"
+
+  variables = {
+    port = "${var.listener_port}"
+  }
+
+  depends_on = "${var.gateway_depends}"
+}
+
+resource "aws_security_group" "service_egress_security_group" {
+  name        = "${var.namespace}-${var.environment}-service_egress_security_group"
+  description = "Allow traffic from service"
```

Comment:
"Allow traffic from service" to do what?

Most of these security group descriptions have been copy/pasted around and aren't actually accurate, and they're a PITA to change later. If we're changing things I'd like to make them useful now, or just ditch them.